tron/core/action.py

```diff
@@ -31,6 +31,13 @@ def state_data(self):
             for field in fields(self)
         }
 
+    def copy(self):
+        kwargs = {
+            field.name: getattr(self, field.name)
+            for field in fields(self)
+        }
+        return ActionCommandConfig(**kwargs)
```

Comment:
i think this should work: `return ActionCommandConfig(**self.__dict__)` or use the property defined just above: `return ActionCommandConfig(**self.state_data)`